src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/MouseInputActionObserver.cs

```diff
@@ -112,25 +112,45 @@ public void ObserveMouse(IEnumerable<ObjectStatus> statefulObjects)
 
         public MouseInputActionData GetCurrentMouseState(Vector2? position = null)
         {
-            var mouse = Mouse.current;
-            if (mouse != null)
+            var pointer = Pointer.current;
```

Comment:
This is our preliminary support for touch/pen pointer events in the current InputSystem during recording/playback.